jekyll/_cci2/caching-strategy.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: classic-docs
-title: "Caching strategies"
+title: "Caching Strategies"
```

Comment:
All titles should be using sentence case inc main page titles. I'm open to changing this though if it's not popular :-D

Reply:
Oh thanks for catching this! I thought I changed this honestly. Do you mean title case? Yes, I think title case makes the most sense for the page titles.

Reply:
To be honest I haven't given it much thought, but the majority of our docs are title case, so I assume when I see sentence case it is a mistake. If you go through and click on random pages in our docs, most of them will be title case. Maybe we can have a discussion about it. Off the top of my head I'm not really sure which I prefer.

Reply:
Yeah we are inconsistent right now but I put all headings to use sentence case in the style guide partly to match up wit marketing, but I do actually prefer it too. I remember Tyler made a script to change them all once so I'm not sure how they have gone wrong, maybe we didn't get round to H1s.